SRC/index.js

```diff
@@ -10,7 +10,13 @@ import routes from './routesTesting/post.routes.test';
 import routeMsgTst from './routesTesting/message.routes.test';
 import routeCmtTst from './routesTesting/comments.routes.test';
 import routeUserTst from './routesTesting/users.route.test';
+import dotEnv from 'dotenv';
+dotEnv.config() 
 
+const mongoose = require('mongoose');
```

Comment:
I'll look through the entire codebase and in our one-on-one, well guide on refactoring to eliminate duplication.